nntrainer/dataset/random_data_producers.cpp

```diff
@@ -54,17 +54,17 @@ class PropsMax : public Property<float> {
  * @brief Props containing number of samples
  *
  */
-class PropsDataSize : public Property<unsigned int> {
+class PropsNumSamples : public Property<unsigned int> {
```

Comment:
Could you elaborate this property? What do you mean by the number of samples?

Reply:
A random data producer has theoretical size. number of samples is used to set theoretical size of the random data producer's data size, so that it can end in indefinite amount of iteration.

Updated comment :)

Reply:
Thanks.